process/interceptors/processor/trieNodeInterceptorProcessor_test.go

```diff
@@ -65,6 +66,43 @@ func TestTrieNodesInterceptorProcessor_SaveShouldPutInCacher(t *testing.T) {
 	assert.True(t, putCalled)
 }
 
+func TestTrieNodeInterceptorProcessor_SignalEndOfProcessingWrongTypeShouldErr(t *testing.T) {
+	t.Parallel()
+
+	cacheMock := &mock.CacherStub{
+		PutCalled: func(key []byte, value interface{}) bool {
+			assert.Fail(t, "should have not arrived here")
+			return false
+		},
+	}
+	tnip, _ := processor.NewTrieNodesInterceptorProcessor(cacheMock)
+
+	intData := interceptedBlocks.InterceptedHeader{}
+	slc := make([]process.InterceptedData, 0)
+	slc = append(slc, &intData)
+	tnip.SignalEndOfProcessing(slc)
```

Comment:
how does this test work, where do you verify that it errors?

Reply:
in PutCalled there is an `assert.Fail()` so it should not arrive there. renamed test